### PR TITLE
Fix name NullPointerException

### LINKED ISSUE #218: Fix name NullPointerException

### PR DESCRIPTION
**Pull request by peironggg**
Fixes #218

### CODE REVIEW

File: src/main/java/seedu/address/scraper/ScraperManager.java [hunk 1]
```diff
@@ -432,7 +431,9 @@ public void getUngradedQuests(WebElement filterBar) {
     private void saveToStorage(List<Mission> missions, List<Quest> quests, List<Student> students, String name)
             throws IOException {
         try {
-            model.setName(name);
+            if (!name.isEmpty()) {
```
Comment (ngzhenteng) [inline src/main/java/seedu/address/scraper/ScraperManager.java:434]: Awesome find, thank you for fixing the bug!

**Review by ngzhenteng — COMMENTED** [review]
LGTM!

File: src/main/java/seedu/address/scraper/ScraperManager.java (hunk 1 above)
Comment (peironggg) [reply src/main/java/seedu/address/scraper/ScraperManager.java:434]: how u leave a review on ur phone

Comment (peironggg) [reply src/main/java/seedu/address/scraper/ScraperManager.java:434]: or u back in ur room alr

**Pull request merged**